public/wp-content/themes/glific/page-contact.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Page contact for theme glific
+ *
+ * @package glific
+ */
+
+	get_header();
```

Comment:
@wevekamola I see the code needs to be formatted a bit. Please update.
Also, there are few plugins that can be used for auto formating. You may want to try this https://packagecontrol.io/packages/phpfmt

Reply:
@Satendra-SR 👍